### PR TITLE
fix(curriculum): changed the description, hint and test of todo app step 6

### LINKED ISSUE #53151: Unclear instructions on Step 6 of "Learn localStorage by building a Todo App"
### Describe the Issue

On Step 6, It is not clear from the instructions that code to remove `hidden` class should be put in the event handler callback.
Also, I do not think that anywhere in exercise before Step 6, `toogle` method on `classList` being used or explained, which can be confusing for new developers that a new method to be used.

**Description:**
Screenshot 2024-01-12 at 8 54 23 PM

<br />
<br />

Instead a much better explanation is provided in the Assertion error

> Your event listener should include a callback function for the second argument using arrow syntax without curly braces. Then you should use the classList property and its toggle method on the taskForm, to toggle the class hidden

<br />

**Error Message:** 
Screenshot 2024-01-12 at 8 54 11 PM

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-localstorage-by-building-a-todo-app/step-6

### Your code

```



```

### Expected behavior

More clear instructions to be provided with explanation to use new `toogle` method on `classList`

### Screenshots

Attached in the description

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Vishakha-Sawra**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53151

Changed the javascript-algorithms-and-data-structures-v8/learn-localstorage-by-building-a-todo-app/step-6 Description, Hint and Test.

### CODE REVIEW

**Review by huyenltnguyen — COMMENTED** [review]
Thank you for the PR.

We will need to highlight keywords, and please also help update the test as it failed CI.

**Comment by Vishakha-Sawra** [comment]
Hi @huyenltnguyen 
Thank you for the comment. I appreciate your guidance. If you have specific instructions on how to resolve the CI test failures, it would be greatly appreciated. Your instructions will help me resolve the issues more effectively.
Thank you!

**Comment by Vishakha-Sawra** [comment]
Hi @huyenltnguyen
Thank you for the comment. I appreciate your guidance. If you have specific
instructions on how to resolve the CI test failures, it would be greatly
appreciated. Your instructions will help me resolve the issues more
effectively.
Thank you!

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md
Comment (JoyShaheb) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md:43]: @gikf  what do you think of this solution? i simplified it a little bit.
```suggestion
assert.match(code, /openTaskFormBtn\.addEventListener\(\s*['"`]click['"`]\s*,\s*\(\s*\)\s*=>\s*{?\s*taskForm\.classList\.toggle\(\s*['"`]hidden['"`]\s*\)\s*;?\s*}?\s*\)\s*;?/)
```

**Review by JoyShaheb — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md
Comment (gikf) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md:43]: My problem with it is allowing mixed up quotes and passing with `{` or `}` just on one side. For example, this would match:

```javascript
openTaskFormBtn.addEventListener(`click", () => {
  taskForm.classList.toggle(`hidden')
);
```

Comment (JoyShaheb) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md:43]: @gikf what would be your simplified solution in this case ? also can you drop your twitter handle ? whould like to say hello

Comment (gikf) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md]: Well, I don't know how to make it simpler, while keeping the correctness, so I wouldn't :slightly_smiling_face: 

I'm not using twitter at this time, you can find me on discord - sanity (or not)

**Review by jdwilkin4 — APPROVED** [review]

**Review by huyenltnguyen — APPROVED** [review]

**Review by JoyShaheb — APPROVED** [review]
looks good, before merging, make sure that all the tests are passing. Thank you for your contribution @Vishakha-Sawra

**Comment by Vishakha-Sawra** [comment]
> looks good, before merging, make sure that all the tests are passing. Thank you for your contribution @Vishakha-Sawra

Hi, @JoyShaheb 
Thank you so much for updating!
Though all the checks have been passed; I cannot merge the PR. Below is the screenshot of what it shows. Can you please guide me about what I can do? 
open-source

**Pull request merged**
